factory/src/lib.rs

```diff
@@ -1,18 +1,21 @@
+use aurora_engine_types::types::Address;
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::serde::{Deserialize, Serialize};
-use near_sdk::serde_json::json;
 use near_sdk::{
-    assert_self, env, ext_contract, near_bindgen, AccountId, Balance, Gas, PanicOnDefault, Promise,
+    assert_self, env, ext_contract, near_bindgen, AccountId, Gas, NearToken, PanicOnDefault,
+    Promise,
 };
 
 const FORWARDER_WASM: &[u8] = include_bytes!("../../res/aurora-forwarder.wasm");
-const INIT_BALANCE: Balance = 1_530_000_000_000_000_000_000_000;
-const STORAGE_BALANCE_BOUND: Balance = 1_250_000_000_000_000_000_000;
-const FORWARDER_NEW_GAS: Gas = Gas(1_000_000_000_000);
+const STORAGE_BALANCE_BOUND: NearToken = NearToken::from_yoctonear(1_250_000_000_000_000_000_000);
+const FORWARDER_NEW_GAS: Gas = Gas::from_tgas(5);
 const MAX_NUM_CONTRACTS: usize = 8;
```

Comment:
Can the value of `MAX_NUM_CONTRACTS` be increased after these changes?

Reply:
I will check it out. However, the number of contracts depends on the amount of gas needed for deploying the forwarder rather than the size of the forwarder contract.

Reply:
Now it's 10.

Reply:
@pustovalov increased to 12 😄